fixes serialization/deserialization of @ symbol

The-Funk opened the pull request.

The-Funk commented:
@baywet updated changelog and got my branch up to date with main. 👍

baywet reviewed (approved):
Thank you for making the changes!

The-Funk commented:
@baywet sorry for the delay, forgot all about this. The symbol cleanup uses PascalCase. I goofed and put "at" as the expectation but "At" should be the correct expectation. Fixed and updated branch.
